src/components/hero/_macro.njk

```diff
@@ -14,8 +14,8 @@
     {% set containerClasses = " hero__container--has-collapsible" %}
 {% endif %}
 
-{# if hero is census specific, change column usage #}
-{% if params.censusTheme is defined and params.censusTheme %}
+{# if hero is census-specific, change column usage #}
```

Comment:
Probably should be if it is census specific or wide page needed, or something like that